src/NUnitFramework/framework/Constraints/DelayedConstraint.cs

```diff
@@ -35,9 +34,159 @@ namespace NUnit.Framework.Constraints
     ///</summary>
     public class DelayedConstraint : PrefixConstraint
     {
+        /// <summary>
+        /// Allows only changing the time dimension of the polling interval of a DelayedConstraint
+        /// </summary>
+        public class WithRawPollingInterval : DelayedConstraint
```

Comment:
Haha, I didn't expect this. But if you find it confusing someone else might as well. Will modify it